src/models/index.ts

```diff
@@ -24,6 +24,7 @@ import * as issue from './issue';
 import * as issueComment from './issue-comment';
 import * as label from './label';
 import * as log from './log';
+import * as locale from './locale';
```

Comment:
I dropped locale on purpose, as this index is used for generating the documentation and `locale` is not an object which the doc generator requires for each model. Could you please revert that? Or are you missing the reference to it anywhere?

Reply:
I am still having 230 lint errors on `platform` with fresh `main`, because of missing `models.locale` declarations, so I've put it back app to launch locally. 

Is part of this update for `platform` released already?

Reply:
Just discussed through chat: we'll accept this change and fix the documentation repo instead.